### PR TITLE
[INLONG-10330][Manager] Support field template management

### LINKED ISSUE #10330: [Improve][Manager] Support field template management
### Description

Support field template management.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #10330

### Motivation

Support field template management.
Users can automatically add field information by selecting a template.
### Modifications

Support field template management.
Divide the visible range of the template into global visibility, visible to responsible parties, and visible to tenants.
The template only allows the responsible person to modify and view based on the visible range.

### CODE REVIEW

**Review by vernedeng — DISMISSED** [review]

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/TemplateVisibleRange.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package org.apache.inlong.manager.common.enums;
+
+/**
+ * TemplateVisibleRange
+ */
+public enum TemplateVisibleRange {
+
+    ALL,
+    IN_CHARGE,
+    TENANT,
```
Comment (aloyszhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/TemplateVisibleRange.java:27]:
```suggestion
    TENANT
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    @ApiModelProperty(value = "Is predefined field, 1: yes, 0: no")
+    private Integer isPredefinedField;
+
+    @ApiModelProperty(value = "Field value for constants")
+    private String fieldValue;
+
+    @ApiModelProperty(value = "Value expression of predefined field")
+    private String preExpression;
+
+    @ApiModelProperty("Is this field a meta field, 0: no, 1: yes")
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java:72]:
```suggestion
    @ApiModelProperty("Is this field a meta field, 0: no, 1: yes, default is 0")
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    private String extParams;
+
+    public TemplateField(int index, String fieldType, String fieldName, String fieldComment, String fieldValue) {
+        this.id = index;
+        this.fieldType = fieldType;
+        this.fieldName = fieldName;
+        this.fieldComment = fieldComment;
+        this.fieldValue = fieldValue;
+        // default
+        this.isMetaField = 0;
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java:99]: `isMetaField` already has the default value 0, plz remove these two lines

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+    public TemplateField(int index, String fieldType, String fieldName, String fieldComment, String fieldValue,
+            Integer isMetaField, String metaFieldName, String originNodeName, String originFieldName) {
+        this(index, fieldType, fieldName, fieldComment, fieldValue);
+        this.isMetaField = isMetaField;
+        this.metaFieldName = metaFieldName;
+        this.originNodeName = originNodeName;
+        this.originFieldName = originFieldName;
+    }
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java:124]:
```suggestion
    public TemplateField(int index, String fieldType, String fieldName, String fieldComment, String fieldValue,
            Integer isMetaField, String metaFieldName) {
        this(index, fieldType, fieldName, fieldComment, fieldValue);
        this.isMetaField = isMetaField;
        this.metaFieldName = metaFieldName;
    }

    public TemplateField(int index, String fieldType, String fieldName, String fieldComment, String fieldValue,
            Integer isMetaField, String metaFieldName, String originNodeName) {
        this(index, fieldType, fieldName, fieldComment, fieldValue, isMetaField, metaFieldName);
        this.originNodeName = originNodeName;
    }

    public TemplateField(int index, String fieldType, String fieldName, String fieldComment, String fieldValue,
            Integer isMetaField, String metaFieldName, String originNodeName, String originFieldName) {
        this(index, fieldType, fieldName, fieldComment, fieldValue, isMetaField, metaFieldName, originNodeName);
        this.originFieldName = originFieldName;
    }
```

**Review by aloyszhang — APPROVED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java (hunk 4 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java]: Fixed.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java (hunk 3 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java]: Fixed.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/TemplateField.java]: Fixed.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/TemplateVisibleRange.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/TemplateVisibleRange.java]: Fixed.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**